share/config.py

```diff
@@ -54,27 +53,13 @@ def __init__(self, output_type: str, kwargs: dict[str, Any]):
         self._dataset: str = ""
         self._namespace: str = ""
```

Comment:
let's remove the assignments above, the setters are invoked below and will take care of initializing the underlying fields